pandas/_libs/tslib.pyx

```diff
@@ -96,6 +96,10 @@ from tslibs.fields import (
     get_date_name_field, get_start_end_field, get_date_field,
     build_field_sarray)
 
+from tslibs.nattype import NaT, nat_strings, __nat_unpickle
+# Note: __nat_unpickle needs to be in the namespace for backward compat pickle
```

Comment:
? why is that, you already do the pickle compat dance.

Reply:
I'd have to go back and check the relevant Travis logs, but there was an error a couple days ago.  Might have been an xarray thing?

Reply:
this needs a rebase anyhow. remove the import and let's see.